### PR TITLE
PR for Migrations and Seeders

### PR DESCRIPTION
**Pull request by shechframgia**
I added migration and seeder for Users and Articles

### CODE REVIEW

File: database/migrations/2016_03_01_062347_create_articles_table.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public function up()
+    {
+        Schema::create('articles', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->text('body');
+            $table->integer('user_id')->unsigned();
+            $table->timestamps();
+            $table->foreign('user_id')
+                  ->references('id')
```
Comment (karlpandacan) [inline database/migrations/2016_03_01_062347_create_articles_table.php]: please check the space here

File: database/seeds/ArticlesTableSeeder.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('articles')->insert([
+            'title' => 'My First Article',
+            'body' => 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+            'user_id' => '1',
+            ]);
```
Comment (jsnulla) [inline database/seeds/ArticlesTableSeeder.php]: `]);` should be inline with `DB:table`

File: database/seeds/UsersTableSeeder.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('users')->insert([
+            'name' => 'shec',
```
Comment (karlpandacan) [inline database/seeds/UsersTableSeeder.php]: same here check the indent here

File: database/seeds/ArticlesTableSeeder.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+        DB::table('articles')->insert([
```
Comment (jsnulla) [inline database/seeds/ArticlesTableSeeder.php]: Please use eloquent

File: database/seeds/ArticlesTableSeeder.php [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('articles')->insert([
+            'title' => 'My First Article',
+            'body' => 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
```
Comment (karlpandacan) [inline database/seeds/ArticlesTableSeeder.php]: Please Use faker.

File: database/seeds/ArticlesTableSeeder.php [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+            'user_id' => '1',
+            ]);
+
+        DB::table('articles')->insert([
+            'title' => 'My Second Article',
```
Comment (karlpandacan) [inline database/seeds/ArticlesTableSeeder.php]: user faker

**Comment by karlpandacan** [comment]
OK :+1:

**Comment by jsnulla** [comment]
OK :+1:

**Pull request merged**